### PR TITLE
Create Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

**Comment by nhirokinet** [comment]
Thank you for translation!
Unfortunately, this did not work on my end. `cn` is country code, and it seems the directory name for Simplified Chinese should be `values-zh-rCN` ?

File: app/src/main/res/values-cn/strings.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <string name="title_for_notification">通知</string>
+    <string name="notFoundOpeningApplicationWithClass">\"%1$s\" 不是一个可启动的包。</string>
+    <string name="calculator_precision_format">计算精度: %1$s</string>
+    <string name="calculator_precision_no_error">无差错</string>
+    <string name="calculator_precision_rounding">进位舍入</string>
+    <string name="calculator_error_division_by_zero">除以 0</string>
+    <string name="summary_text_for_preferences_of_this_app">Blue Line Console 选项</string>
+    <string name="notification_title_on_startup">关于“help” 命令</string>
+    <string name="notification_detail_on_startup_pre">要显示用法，输入</string>
+    <string name="notification_detail_on_startup_main" translatable="false">help</string>
```
Comment (nhirokinet) [inline app/src/main/res/values-cn/strings.xml:18]:
```suggestion
```
Can you remove this line as untranslatable strings in non-default language is duplicated code, and is reported as error in Android Studio?

**Comment by sr093906** [comment]
@nhirokinet Done. Also impemented changes made in https://github.com/tachiyomiorg/tachiyomi/compare/tachiyomiorg:d11c72f...tachiyomiorg:444cefc

**Comment by nhirokinet** [comment]
Thank you!

**Comment by nhirokinet** [comment]
Released in1.0.10

**Pull request merged**